docs/OV_Runtime_UG/performance_hints.md

```diff
@@ -132,4 +132,4 @@ The `benchmark_app`, that exists in both  [C++](../../samples/cpp/benchmark_app/
  
 
 ### See Also
```

Comment:
```suggestion
### Additional Resources
```
It seems that Github does not allow me to suggest the same change in multi_device.md. Could you have a look into that, thanks :)

Reply:
Change for multi_device commited in https://github.com/openvinotoolkit/openvino/pull/13551/commits/e7f3cd118aa3f46f1862fb5f2a0862ebf072e2ea